nengo_loihi/tests/test_simulator.py

```diff
@@ -412,38 +412,74 @@ def test_double_run(precompute, Simulator, seed, allclose):
     assert allclose(sim1.data[probe], sim0.data[probe])
 
 
-def test_simulator_noise(request, plt, seed):
+@pytest.mark.parametrize('exp', [-4.5, -3, -2])
```

Comment:
Is there any particular significance to these noise exponents? Like are -2 and -3 testing different things?  And do you know what these translate to in the discretized range (specifically, if we're testing exponents <7 and >7?).

Reply:
Yeah, I did print out what they translate to, it's 5, 10, and 13. Basically I chose one to be discretized under 7, one to match the value that we were typically using for noise for our interneurons (-2) when we used noisy ones, and one in between. If you use too large of a value (e.g. -1) then some of the neurons may start firing, which messes up the statistics. We'd probably be fine with just -4.5 and -2, if we want to save on a test.

Reply:
Those values make sense to me then.  Maybe just add a quick comment noting what they translate to?